### ISSUE TITLE
Harbor ui does not show tags pushed to the registry

### ISSUE THREAD

**Issue by jakedsouza** [vmware/harbor#2207]
If you are reporting a problem, please make sure the following information are provided:
1)Version of docker engine and docker-compose.
```
Version 17.05.0-ce-rc1-mac8 (16582)
Channel: edge
73d01bb48e
```
2)Config files of harbor, you can get them by packaging "harbor.cfg" and files in the same directory, including subdirectory.
Deployed harbor using the VIC OVA. 

3)Log files, you can get them by package the /var/log/harbor/ .
[harbor.zip](https://github.com/vmware/harbor/files/974243/harbor.zip)
<img width="1149" alt="screen shot 2017-05-03 at 11 01 34" src="https://cloud.githubusercontent.com/assets/1404569/25674514/ece4b7c8-2fef-11e7-8a7e-72bb08763533.png">
<img width="1144" alt="screen shot 2017-05-03 at 11 01 28" src="https://cloud.githubusercontent.com/assets/1404569/25674513/eccfd736-2fef-11e7-8b1d-89a7e151542a.png">

**Comment by steven-zou** [comment]
@jakedsouza 

  Thanks for raising the issue. We'll take a look.

@wknet123 
  Please take a look at what's the problem.

**Comment by wknet123** [comment]
It is a known issue in release-1.1.1 which has been fixed in release-1.1.1-rc3 with PR #2201.

**Comment by steven-zou** [comment]
Close this issue as it was fixed.
